src/win32.mak

```diff
@@ -194,7 +194,9 @@ RUN_BUILD=$(GEN)\build.exe --called-from-make "OS=$(OS)" "BUILD=$(BUILD)" "MODEL
 
 defaulttarget: $G debdmd
 
-auto-tester-build: $G dmd checkwhitespace
+auto-tester-build: $(GEN)\build.exe
+	$(RUN_BUILD) "DDEBUG=" "ENABLE_RELEASE=1" $@
+	copy $(TARGETEXE) .
```

Comment:
Why is copying the final exe necessary? And also why is it necessary to explicitly pass those flags in the previous step?

Reply:
Previously the dmd was built by 

> Why is copying the final exe necessary?

I don't like this either but the test suite uses `dmd/src/dmd.exe`

> And also why is it necessary to explicitly pass those flags in the previous step?

The executable was previously built by `reldmd` which invokes `make` recursively and overrides these flags. Otherwise `dmd.exe` would be built with `-unittest -cov` enabled.

Reply:
Thanks for the explanations!

> I don't like this either but the test suite uses dmd/src/dmd.exe

Just like the posix.mak files don't expect dmd to be available at this path for the test suite, we should fix the Win32.mak files as well. This obviously outside of the scope of this PR, so I suggest leaving a FIXME comment explaining the situation.

> The executable was previously built by reldmd which invokes make recursively and overrides these flags. Otherwise dmd.exe would be built with -unittest -cov enabled.

I think this should be fixed as well (perhaps in a future PR). 
Like other targets that build dmd, whether it's build in release mode or debug + unittest, should be configurable based to env variables like `ENABLE_RELEASE`. We should run the test suite with both asserts enabled and disabled.

Reply:
You're welcome. I've added `FIXME`s for both issues and removed the `ENABLE_RELEASE` override (`DDEBUG` is depreacted anyway).

This implies a slight change as the test suite is now run with bound-checks and asserts instead of optimizations but we can add it as an additional run 'from the outside' .

Reply:
Yes, what I meant that whether optimizations are enabled or disabled should be disabled from outside.

Reply:
Ahh, then I'll would suggest keeping the current behaviour and changing it in another PR